modules/ROOT/pages/Certificates.adoc

```diff
@@ -1,58 +1,54 @@
 = Certificate Settings
 
-All certificates in Anypoint Partner Manager are securely stored in Anypoint Platform's secrets manager, which uses a secure vault technology to store and control access to private keys, passwords, certificates, and other secrets.
+All certificates in Anypoint Partner Manager are securely stored in Anypoint Platform’s secrets manager, which uses secure vault technology to store and to control access to private keys, passwords, certificates, and other secrets.
 
-Configure your SSL certificate settings based on your certificate needs.
+You can configure your SSL certificate settings based on your certificate needs.
 
 The information provided for each certificate includes:
 
-* *Authority* +
-Certificate provider
-* *Serial Number* +
-Serial number from the certificate
-* *Status* +
-Active is shown with a green icon or Inactive is shown with a red icon. +
-* *Start Date* +
-Date from when the certificate is valid
-* *Expires* +
-Date after which the certificate is no longer valid 
-* *Endpoints* +
-Number of endpoints where the certificate is used
-* *Message Flows* +
-Number of message flows that use the certificate
+* *Authority*: The certificate provider
+* *Serial Number*: The serial number from the certificate
+* *Status*: The status of the certificate. The color coded icon on the certificate shows the user the status of the certificate:
+** Green: The certificate is active
+** Red: The certificate is inactive
+* *Start Date*: The date from when the certificate is valid
+* *Expires*: The date after which the certificate is no longer valid
+* *Endpoints*: The number of endpoints using the certificate
+* *Message Flows*: Number of message flows using the certificate
 
 In Partner Manager, you can:
```

Comment:
This is a very vague introduction. Can we reword to somehow introduce the precise information following: certificate types and connection settings?

Reply:
Done - added heading